l10n_it_location_nuts/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,20 @@
+**Italiano**
+
+Dopo aver installato il modulo, aprire il wizard:
+
+Vendite / Configurazione / Contatti / Localizzazione / Import NUTS 2013
```

Comment:
Il wizard si trova in Contatti / Configuration / Localizzazione / Import NUTS 2013

Reply:
Fatto, grazie!